corehq/apps/export/views/new.py

```diff
@@ -412,6 +413,21 @@ def create_new_export_instance(self, schema, username, export_settings=None):
     def get(self, request, *args, **kwargs):
         case_type = request.GET.get('export_tag').strip('"')
 
+        if (
+            case_type not in get_case_types_for_domain(request.domain)
+            and case_type != ALL_CASE_TYPE_EXPORT
+        ):
+            messages.error(
+                request,
+                _(
+                    "Case type '%(case_type)' does not exist in domain %(domain)"
```

Comment:
Just a question for myself. Back in my product days, it seemed like we tried to avoid using the word "domain" in user facing text, but I'm not sure if that is still the case/if we ever really followed that strictly.

Reply:
That does sound like a good practice. I can reword it as  - 
> Case type {case_type} does not exist in project space

Does that look okay to you?

Reply:
https://github.com/dimagi/commcare-hq/pull/35035/commits/6e2b364d9c1e9503a6e24b62e435259a1425b6c1